src/osdc/Objecter.h

```diff
@@ -1200,6 +1202,8 @@ class Objecter : public md_config_obs_t, public Dispatcher {
   MonClient *monc;
   Finisher *finisher;
   ZTracer::Endpoint trace_endpoint;
+  dmc::ServiceTracker<int> *qos_trk;
```

Comment:
could use `unique_ptr<>`

Reply:
Yes, that would be preferable for clean-up.

Reply:
Yes, used unique_ptr<> as you said.